content/dstu2/condition.md

```diff
@@ -125,6 +147,10 @@ To successfully PUT a condition, the following headers must be provided. Conditi
     Authorization: <OAuth2 Bearer Token>
     If-Match: W/"<Current condition id>"
 
+_Implementation Notes_
+
+* Currently, `health-concern` category code from the [argonaut] system is not supported for updating conditions.
```

Comment:
Just call out that health-concern is not supported for update. No need to call out the system. It's not ambiguous.